taichi/ir/ir.h

```diff
@@ -387,6 +387,24 @@ class StmtFieldManager {
   mark_fields_registered(); \
   io(field_manager)
 
+struct Location {
+  int line_number;
+  std::string var_name;
+};
+
+struct DebugInfo {
+  Location src_loc;
+  std::string tb;
+
+  explicit DebugInfo() = default;
```

Comment:
nit: usually constructor with exactly 1 single argument should be marked as "explicit" to avoid implicit conversion. Therefore, the following two DebugInfo(std::string...) are better targets for "explicit" decorator